html-syntax/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Syntax</title>
+  </head>
+
+  <body>
+    <img src="https://assets.pokemon.com/assets/cms2/img/pokedex/full/025.png">
```

Comment:
Make sure to add the `alt` attribute with a suitable description of the image.

Reply:
Thanks so much! I got the bot message at work and went, "uh oh the alt". I'll fix it now